### DISCUSSION TITLE
Foregound Thread에서 데이터 버퍼란?

### DISCUSSION THREAD

**Discussion by CODe5753** [Well-Founded-Confidence/RealMySQL#8] [Q&A]
Foreground Thread는 데이터 버퍼나 캐시로부터 데이터를 가져온다고 해요.
이 문장을 봤을 때, 캐시와 또 다른 개념인 것 같은데, 어떤 개념일까요?

**Comment by menthamin** [comment]
MyISAM 키캐시: 데이터와는 다르게 해석
- 데이터 버퍼가 있는데 사용하지 않는 것 같다.
InnoDB 버퍼풀: 데이터를 캐싱하는 것
